### DISCUSSION TITLE
does not exist in your theme config. 'spacing' has the following keys

### DISCUSSION THREAD

**Discussion by tuannda** [tailwindlabs/tailwindcss#8651] [Help]
`utilities.css 'spacing[some key][1.5]' does not exist in your theme config. 'spacing' has the following keys: '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '14', '16', '20', '24', '28', '32', '36', '40', '44', '48', '52', '56', '60', '64', '72', '80', '96', 'px', '0.5', '1.5', '2.5', '3.5'
`

I just has error above, Please help me fix it.

```
"tailwindcss": "^3.1.3",
```

**Comment by rohmann** [comment]
Hi @tuannda what is the class you're using that generates this error?

**Reply by tuannda** [reply]
I'm sorry, but I don't know it. You can pull this repo and help me (you don't need install web wordpress, only npm install && npm run dev)

https://github.com/tuannda/basewp/tree/main/wp-content/themes/basewp

**Reply by rohmann** [reply]
I see. I can't pull all that down at troubleshoot at the moment but after a quick look, you do have this in your config

```
spacing: {
            'some key': {
                1.5: '0.375rem'
            }
        },
```

Try removing that, or updating it to

```
spacing: {
  '1.5': '0.375rem'
}
```

You may also want to put that inside `extend` instead of directly under `theme` or you'll lose all the original values.

**Reply by tuannda** [reply]
Thank @rohmann so much. :)

**Comment by crswll** [comment]
Tailwind is likely looking inside `node_modules` due to your `content` array in your `tailwind.config.js`. Make sure the globs in `content` aren't looking inside `node_modules`. Check out [Pattern Recommendations](https://tailwindcss.com/docs/content-configuration#pattern-recommendations) in the Content Configuration part of the docs.

**Reply by tuannda** [reply]
Oh, thank @crswll  so much. I just modify it. I fixed and I see webpack build very fast.
